ceresdb-protocol/src/main/java/io/ceresdb/options/CeresDBOptions.java

```diff
@@ -472,18 +413,12 @@ public CeresDBOptions build() {
             opts.writeOptions = new WriteOptions();
             opts.writeOptions.setMaxWriteSize(this.maxWriteSize);
             opts.writeOptions.setMaxRetries(this.writeMaxRetries);
-            opts.writeOptions.setMaxInFlightWriteRows(this.maxInFlightWriteRows);
+            opts.writeOptions.setMaxInFlightWritePoints(this.maxInFlightWriteRows);
```

Comment:
Keep the parameters and arguments consistent？
```suggestion
            opts.writeOptions.setMaxInFlightWritePoints(this.maxInFlightWritePoints);
```

Reply:
fixed